internal/capability/operator_install.go

```diff
@@ -9,37 +9,37 @@ import (
 
 var logger = log.Sugar
 
-func (ca *CapAudit) OperatorInstall() error {
-	logger.Debugw("installing package", "package", ca.Subscription.Package, "channel", ca.Subscription.Channel, "installmode", ca.Subscription.InstallModeType)
+func (ca *capAudit) OperatorInstall() error {
+	logger.Debugw("installing package", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
 
 	// create operator's own namespace
-	operator.CreateNamespace(context.Background(), ca.Namespace)
+	operator.CreateNamespace(context.Background(), ca.namespace)
```

Comment:
Does this return an error?

Reply:
Yes it actually does! Good catch! Here it is 39d5f16